ephios/core/signals.py

```diff
@@ -86,6 +115,19 @@
 will contain a list of event ids on which the action should be performed.
 """
 
+register_event_action = PluginSignal()
+"""
+This signal is sent out to get a list of actions that a user can perform on a single event. The actions are
+displayed in the dropdown menu on the event detail view. Each action is represented by a dict with the keys
+``url``, ``label`` and ``icon``. The given url will be called with a ``pk`` parameter containing the id of the event.
+"""
+
+homepage_info = PluginSignal()
+"""
+This signal is sent out to get additional information to display on the homepage.
+Receivers receive a ``request`` keyword argument. Receivers should return html that is rendered inside a card.
+"""
```

Comment:
Do we want to wait with this until we figured out a nicer HomeView design? Or do you have a usecase in mind right now?